picard.spec

```diff
@@ -28,9 +29,17 @@ def get_locale_messages():
     return data_files
 
 
-
 block_cipher = None
 data_files = get_locale_messages()
+os_name = platform.system()
+fpcalc_name = 'fpcalc'
+
+if os_name == 'Windows':
+    fpcalc_name += '.exe'
+
+if os.path.isfile(fpcalc_name):
+    data_files += [(fpcalc_name, fpcalc_name)]
```

Comment:
Not sure it such program should go in datas or binaries: https://pythonhosted.org/PyInstaller/spec-files.html#spec-file-operation
How does it handle file perms ? (on systems like macosx)

Reply:
Also second argument of the tuple is supposed to be a folder:
https://pyinstaller.readthedocs.io/en/stable/spec-files.html#adding-data-files
"The second specifies the name of the **folder** to contain the files at run-time."